ports/cuda-api-wrappers/portfile.cmake

```
@@ -0,0 +1,56 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO eyalroz/cuda-api-wrappers
+    REF "v${VERSION}"
+    SHA512 4bc095513ed1a40f7239810abf7f6edcfde5471a89de8cf27a76038f6a54f6234542693bb606cc5e389403f3d12cb186b5a9cfb31c2bf3e437c112d215fb872d
+    HEAD_REF master
+)
+
+# cuda toolkit check
+vcpkg_find_cuda(OUT_CUDA_TOOLKIT_ROOT CUDA_TOOLKIT_ROOT)
+message(STATUS "CUDA_TOOLKIT_ROOT ${CUDA_TOOLKIT_ROOT}")
+
+# windows nvcc compiler path
+if (WIN32)
+    set(CMAKE_CUDA_COMPILER "${CUDA_TOOLKIT_ROOT}/bin/nvcc.exe")
+endif()
+
+# unix nvcc compiler path
+if (UNIX)
+    set(CMAKE_CUDA_COMPILER "${CUDA_TOOLKIT_ROOT}/bin/nvcc")
+endif()
```

```suggestion
set(CMAKE_CUDA_COMPILER "${CUDA_TOOLKIT_ROOT}/bin/nvcc${VCPKG_HOST_EXECUTABLE_SUFFIX}")
```
or just put the expression after `-DCMAKE_CUDA_COMPILER=` below.